metabase/metabase
# Adding snippets back into getCollection

Closes https://github.com/metabase/metabase/issues/44092

### Description
We recently made changes to drive collections using RTK Query, where we changed `getCollection` to simply accept an id. However, `snippet-collections` are driven from collections and require passing a namespace to generate a correct URL. This ended up being a problem in a very specific scenario where a user doesn't have access to `Our Analytics` and wants to interact with snippets

### How to verify
1. Create some snippets
2. Log in as a user with no view access to `Our Analytics`
3. New -> SQL Query -> open snippets sidebar. You should be able to interact with snippets

### Checklist
